CMakeLists.txt

```diff
@@ -339,10 +340,19 @@ else()
 	set(HAVE_FCNTL_H 1)
 	set(HAVE_UNISTD_H 1)
 	set(HAVE_STDINT_H 1)
+	set(HAVE_STDBOOL_H 0)
 	set(HAVE_INTTYPES_H 1)
 	set(HAVE_SYS_FILIO_H 1)
 endif()
 
+if(HAVE_STDBOOL_H)
+	add_definitions(-DHAVE_STDBOOL_H)
+endif()
+
+if(HAVE_STDINT_H)
+	add_definitions(-DHAVE_STDINT_H)
+endif()
+
```

Comment:
@bmiklautz didn't want to include ```config.h``` in ```wtypes.h``` which would trigger a complete rebuild after every change in ```config.h```